src/components/orderDetails/OrderDetails.tsx

```diff
@@ -0,0 +1,299 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+// import {
+//   URL_OMS_ORDER_STATUS,
+//   URL_ORDER_DETAILS,
+// } from "../../constants/apiConstants";
+import omsOrderStatusJSON from "../../sampleJSON/OmsOrderStatus.json";
+import orderDataJSON from "../../sampleJSON/orderData.json";
+
+import {
+  AMOUNT,
+  ATG,
+  BITIER,
+  CHANNEL,
+  CUSTOMER_INFO,
+  EMAIL,
+  ITEMS_INFO,
+  LOCALE,
+  NAME,
+  NO_MATCHING_ORDERS_FOUND,
+  OMS,
+  ORDER,
+  ORDER_DETAILS,
+  ORDER_TOTAL,
+  ORDER_TYPE,
+  PAYMENT_INFO,
+  PAYMENT_TYPE,
+  PROMOTIONS,
+  STATUS,
+  STATUS_ACROSS,
+  SUBMITTED,
+  WMS,
+} from "../../constants/appConstants";
+import {
+  CommerceItem,
+  CommerceItemData,
+  OmsOrderStatus,
+  OrderData,
+} from "../../@types/OrderDetails";
+// import { fetchData } from "../../utils/fetchUtil";
+import Table from "../common/Table";
+import CustomIcon from "../common/CustomIcon";
+import Card from "../common/Card";
+import PromotionsIcon from "../../assets/promotions.svg";
+import OrderClockIcon from "../../assets/order_clock.svg";
+import OmsInfoIcon from "../../assets/oms_info.svg";
+import Loader from "../loader";
+
+const OrderDetails: React.FC = () => {
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [orderData, setOrderData] = useState<OrderData>({});
```

Comment:
You would have got a typescript warning in this line.

Reply:
added {} as well